ubuntu/releases.go

```diff
@@ -121,6 +124,15 @@ var focalDist = &claircore.Distribution{
 	VersionCodeName: "focal",
 }
 
+var impishDist = &claircore.Distribution{
+	Name:            "Ubuntu",
+	Version:         "21.10 (Impish Indri)",
+	DID:             "ubuntu",
+	PrettyName:      "Ubuntu 21.10",
```

Comment:
Guessing at this based on past distos, `Ubuntu Impish Indri (development branch)` is currently what I've seen